app/components/submission-title-cell/component.js

```diff
@@ -1,6 +1,20 @@
 import Component from '@ember/component';
 import { alias } from '@ember/object/computed';
+import { isArray } from '@ember/array';
 
-export default Component.extend({
-  submission: alias('record')
-});
+/**
+ * Named properties passed to this component:
+ *  - record {Submission}
+ */
+export default class SubmissionTitleCell extends Component {
+  @alias('record') submission;
```

Comment:
This might need to be:
```suggestion
  @alias('args.record') submission;
```